spyrmsd/parallel.py

```diff
@@ -0,0 +1,118 @@
+try:
+    from pebble import ProcessPool
+except ImportError:
+    errmsg = (
+        "Parallel execution of SPyRMSD (`prmsdwrapper`) requires `pebble`."
+        "Please install `pebble`: https://github.com/noxdafox/pebble"
+    )
+    raise ImportError(errmsg)
+
+import os
+from concurrent.futures import TimeoutError
+from functools import partial
+from typing import List, Optional, Union
+
+import numpy as np
+
+from spyrmsd import molecule
+from spyrmsd.rmsd import rmsdwrapper
+
+
+def prmsdwrapper(
+    molrefs: Union[molecule.Molecule, List[molecule.Molecule]],
+    mols: Union[molecule.Molecule, List[molecule.Molecule]],
+    num_workers: Union[int, None] = 1,
```

Comment:
For usability, I think it might be worth keeping the arguments of `prmsdwrapper` the same as `rmsdwrapper`, therefore I'd move this as the first new argument (before `timeout` and `chunksize`).

Reply:
```suggestion
    num_workers: Optional[int] = 1,
```
?

And maybe it's worth setting it to `None` by default, what do you think?